### PR TITLE
Update the Community Contributors list

### PR DESCRIPTION
**Pull request by marcostudios**
I don't know if it should include the Elite Nominators or if it should be on another page

### CODE REVIEW

File: wiki/People/Community_Contributors/en.md [hunk 1]
```diff
@@ -142,6 +142,36 @@ Not to be mistaken as [osu! Alumni](/wiki/People/osu!_Alumni), which are previou
...
+| ![SE][flag_SE] [PuffBuck](https://osu.ppy.sh/users/4234525) | Outstanding contribution to the osu! World Cups & Twitch moderation |
+| ![GB][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | Outstanding contribution to the osu! community, the tournaments scene and excellence as a role model |
+| ![AT][flag_AT] [Omgforz](https://osu.ppy.sh/users/578943) | Outstanding contribution to the osu! World Cups & commentary/casting excellence |
+| ![AU][flag_AU] [Kano](https://osu.ppy.sh/users/3036203) | Outstanding contribution to the osu! World Cups & commentary/casting excellence |
+| ![US][flag_US] [Halogen-](https://osu.ppy.sh/users/169992) | Outstanding contribution to the osu!mania tournament scene |
+| ![DE][flag_DE] [Junihuhn](https://osu.ppy.sh/users/4182339) | Outstanding contribution to the osu! World Cups via video recap creation and other assistance |
+| ![NL][flag_NL] [Sartan](https://osu.ppy.sh/users/4100941) | Outstanding contribution to the osu! World Cups & commentary/casting excellence |
+| ![RU][flag_RU] [Kobold84](https://osu.ppy.sh/users/3227533) | Outstanding moderation of the Russian language-specific channel |
```
Comment (DoryanR) [inline wiki/People/Community_Contributors/en.md]: > ![RU][flag_RU] [Kobold84](https://osu.ppy.sh/users/3227533)

On my side the image of the flag is not displayed, however I would not propose a solution to remedy it, I don't know enough about it.

**Comment by marcostudios** [comment]
It should be there now

**Comment by DoryanR** [comment]
No, for me I still don't see it, both on my editor and on the site with the rich diff, I always see "![RU][flag_RU]"

File: wiki/People/Community_Contributors/en.md [hunk 2]
```diff
@@ -168,6 +198,7 @@ Not to be mistaken as [osu! Alumni](/wiki/People/osu!_Alumni), which are previou
 [flag_NZ]: /wiki/shared/flag/NZ.gif "New Zealand"
 [flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
 [flag_PL]: /wiki/shared/flag/PL.gif "Poland"
+[flag RU]: /wiki/shared/flag/RU.gif "Russian Federation"
```
Comment (Joehuu) [inline wiki/People/Community_Contributors/en.md]: This should fix it.
```suggestion
[flag_RU]: /wiki/shared/flag/RU.gif "Russian Federation"
```

Comment (marcostudios) [reply wiki/People/Community_Contributors/en.md]: Thank you

**Pull request merged**